### ISSUE TITLE
[Bug]: SbbDialog content is barely visible in strong zoom levels (A11y)

### ISSUE THREAD

**Issue by httpxeile** [sbb-design-systems/lyne-components#5267]
### Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/4) for an issue that matches the one I want to file, without success.

### Bug type

Accessibility

### It affects the following packages

Components

### Which version of Lyne Components are you using

5.8.0

### Operating system

macOS 26

### Browser / Browser version

Google Chrome 153.0.8010.48

### Input mode

Mouse/Keyboard

### Additional settings

400% Zoom

### What happened?

**Current behaviour**
When increasing the zoom level on smaller screens (1280px width) up to 400%, the dialog title and dialog actions grow. The remaining space for the dialog content is getting smaller with higher zoom levels. At the zoom level of 400% the dialog content is presented in a very small area. Thus, it makes it hard for the user to navigate the dialog content or perform inputs.

<img width="1506" height="832" alt="Image" src="https://github.com/user-attachments/assets/318d60ff-7330-4ec9-aecc-63fb1db392f0" />
<img width="1503" height="825" alt="Image" src="https://github.com/user-attachments/assets/e697813e-fb36-4494-afe9-540bf887aeb4" />

**Expected behaviour** 
In higher zoom levels, the dialog reserves enough space (e.g. 1/3 of the dialog or so) to display the content in an accessible way, enabling the user to navigate the content.

**Affected components**
SbbDialog

### Steps to reproduce the issue

1. Open https://www.sbb.ch/en/offer?reduction=hta123&tripId=3HA.eNpVkttu2kAQhkXStBK9803VSJWsGFWVrJSd9TkSEraxwRgw2I453FScIUAgYAOO8jB9Cq56RR-hUl-nXRKStLPa2fl399vRrCb5-4z6dbbfFWxrv_NTagayTqb188dy3B3RBS3byMgCjzBAtpnhJU6SQZKyJbKJOIRQtp0BLGePmBd-pfPt6bR_SzCFExVZkR4xHnMvmMgh_IxhhEUAMkUOvwqJCEunBRoELKVSkHq0_c7W9zsmMBjM6CYDCDG6yhBXJs6zdJ8BRi0zAIKAZHSIMTlQfaRWGCfHMK7PSMzhkeMriDHcPGEKVuUQ6zZDShIE_PC6iPCyIDJAFB4POeUBiOQB-Mf9SwykRJkTefmQwH3OAER5JIZvBX5lqU-mqxhXG4jNlTXfXMuhsr27Nl24H7SMcdwK8nq1OGwWhKXrX8cjnEuXdCcv9eOp6w388DasDiIfuKG1rm0sPaqMDcOy-fq24XhLu9iZmgtZHUXD3jXKdzQo1zZ2WbP1rt-ZBLHVnNzok0ozrc3qft_36o5o1bxwoiib2qYyZwsNaTBs5bsDYc6ibXtQy2FnngujeOaZSF5oo0Xg15dcZARbvlcSep5XbqpbFlejuFRtp-VytWSsmsALUVqrsh0WayNeWjilOdg9tQe3-moYiatNf9mdGYO8ctPC6zi9iZbrIN3RYldWFcm8aYbeTKs7rBksZ7W1uG0XDaE3U9YdWww6aNwohq1Ws-r4vTysWqwgB64Q3N9rd-aoUO6ul9NwYd7xGvnnDPU9QaWS746NSn08tNclwCUWfRCvOHyFEIuAeOr98RaIdOIoRILQJ9TnJ0E69vw_XvqXp0_PPyRPLJ1KCOdvDi178ZbQCACdXnxKJlTqxKXpP8-W-PIa_gWa_gE2.eNqrVirLTCxWsoqO1VHyV7JSsjA1MDYwMFDSUXKB8MyMDYyAvBQgz8jAyEzX0FDXyAwoUAIUMDSzMrQEshOBbD8gnQOkHYF0npJVXmlOTi0Ay_MU0Q
2. Set the window / screen size to approx. 1280 × 720 pixels
3. Set the zoom level to 400%
4. Click "Seat reservation" button
5. Tick the checkbox
6. Click "Select seat"
7. The dialog content is not accessible

This is independent from the dialog content. Here is another example:
1. Open https://www.sbb.ch/en/offer
2. 2. Set the window / screen size to approx. 1280 × 720 pixels
3. Set zoom level to 400%
4. Click "Purchase as a guest" button
5. The dialog content is hardly accessible, but scrollable

### Relevant log output

```shell

```

**Comment by kyubisation** [comment]
Hello there

The WCAG states that the minimum height that needs to be supported is 256px, meaning 1024px at 400%.
https://www.w3.org/WAI/WCAG21/Understanding/reflow.html

I will try to improve the situation a little bit, by reducing the blocked area at the top, but the rest is up to you.
Strategies you can attempt are:

- Use a lower visual level for `<sbb-dialog-title>`
- Replace usage of `<sbb-dialog-actions>` with buttons in the content
- Use size s for the button in `<sbb-dialog-actions>`

Also please note that the current state where you cannot scroll is a problem on your part, as when I remove `height: 100%;` from `.h-full` I can scroll the content.
